### PR TITLE
[Static Route Expiry] Feature support

### PR DESCRIPTION
**Pull request by sumukhatv**
This PR has two parts:
  1. Support for two new APIs:
      GET
      Path: `/v1/config/vrf/static_route/route_expiry`
      Return value: `{"time": <time in seconds>}`
      
      POST
      Path: `/v1/config/vrf/static_route/route_expiry`
      Param: `{"time": <time in seconds>}`

 2. Modification to VRF route PATCH API:
     - check if static route is persistent or not and add it to the right DB

### CODE REVIEW

File: sonic_api.yaml [hunk 1]
```diff
@@ -1512,7 +1512,22 @@ paths:
...
+      description: This API call receives a list of route entries to be added/modified/deleted from the virtual
```
Comment (prsunny) [inline sonic_api.yaml:1515]: Why is this changed?. It should be a new API for expiry, correct?

Comment (sumukhatv) [reply sonic_api.yaml:1515]: This has not changed. I just broke it down to new lines so it can be easily read.

File: go-server-server/go/default.go [hunk 2]
```diff
@@ -1445,6 +1456,9 @@ func ConfigVrfVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
             if r.Profile != "" {
                 route_map["profile"] = r.Profile
             }
+            if r.Persistent == "false" {
```
Comment (prsunny) [inline go-server-server/go/default.go:1459]: At line 1434, it will continue the operation without setting if the route is identical. So how do you reset this `route_map["refresh"] `attribute when client refreshes the route? Can you please add the test?

File: go-server-server/go/default.go [hunk 3]
```diff
@@ -1477,20 +1532,32 @@ func ConfigVrfVrfIdRoutesGet(w http.ResponseWriter, r *http.Request) {
...
-
+    persistent = false
+    pattern = generateDBTableKey(db.separator, STATIC_ROUTE_TB, vrf_id_str, ipprefix)
     kv, err := GetKVsMulti(db.db_num, pattern)
     if err != nil {
         WriteRequestError(w, http.StatusInternalServerError, "Internal service error", []string{}, "")
         return
     }
 
+    if len(kv) == 0 {
```
Comment (prsunny) [inline go-server-server/go/default.go:1548]: I think for "GET", we should fetch both app_db and config_db. Client can identify it based on "persistent" field.

File: test/test_restapi.py [hunk 4]
```diff
@@ -1162,6 +1162,99 @@ def test_vrf_routes_all_verbs(self, setup_restapi_client):
...
+        routes.append({'cmd':'add',
+                            'ip_prefix':'70.1.2.0/24',
+                            'nexthop':'192.168.2.200,192.168.2.201,192.168.2.202',
+                            'nexthop_monitor':'192.168.3.200,192.168.3.201,192.168.3.202',
+                            'weight':'10,20',
+                            'profile':'profile1',
+                            'persistent': 'true'})
+
+
+        # Patch add
```
Comment (prsunny) [inline test/test_restapi.py:1203]: Suggest please add a unit-test for refresh field. Also, please have one or two cases with IPv6 entries as that is the main use-case

File: go-server-server/go/default.go (hunk 3 above)
Comment (sumukhatv) [reply go-server-server/go/default.go:1548]: We are checking both DBs here. The GET API does not take any parameters. Since mostly our use-case is non-persistent, I check the APP DB first and then if I can't find anything, I check the CONFIG DB. Since the VRF name is going to be unique, we should be good here.

**Review by prsunny — APPROVED** [review]
